src/Microsoft.FeatureManagement.AspNetCore/UseForFeatureExtensions.cs

```diff
@@ -22,20 +22,14 @@ public static class UseForFeatureExtensions
         /// <returns></returns>
         public static IApplicationBuilder UseForFeature(this IApplicationBuilder app, string featureName, Action<IApplicationBuilder> configuration)
         {
-            if (app == null)
-            {
-                throw new ArgumentNullException(nameof(app));
-            }
+            ArgumentNullException.ThrowIfNull(app);
 
             if (string.IsNullOrEmpty(featureName))
```

Comment:
Confused why it would suggest Argument.ThrowIfNull for null checks, but not ThrowIfNullOrEmpty for string checks.

Reply:
I had the same thought. Turns out ThrowIfNull was added in .NET 6 while ThrowIfNullOrEmpty was added in .NET 7. And we still have .NET 6 as a target.